diagrams/lib/main.dart

```diff
@@ -26,7 +27,8 @@ import 'tile_mode.dart';
 
 Future<Directory> prepareOutputDirectory() async {
   final Directory directory = new Directory(
-    path.join((await getApplicationDocumentsDirectory()).absolute.path, 'diagrams'),
+    path.join(
+        (await getApplicationDocumentsDirectory()).absolute.path, 'diagrams'),
```

Comment:
missing trailing comma for better styling

Reply:
Fixed.